snoop/migrations/0004_auto_20161004_0825.py

```diff
@@ -68,6 +68,21 @@
     path = '' and filename = '' and content_type = 'application/x-directory'
 """
 
+def count_documents(apps, schema_editor):
```

Comment:
Judging by the way you're using this function, you can call it `documents_exist`, and it can return a boolean.